src/AppInstallerCommonCore/PortableARPEntry.cpp

```diff
@@ -62,6 +62,7 @@ namespace AppInstaller::Registry::Portable
         }
         else
         {
+            m_exists = false;

```

Comment:
m_exists should be initialized to false in the header

Reply:
add initialization to header

Reply:
nit: that would mean you wouldn't need to do it here.